en/02_Developer_Guides/17_CLI/02_Hybrid_Commands.md

```diff
@@ -0,0 +1,172 @@
+>
+> See [the symfony/console colouring documentation](https://symfony.com/doc/current/console/coloring.html) for information about styling output. Note that all output (including outputting for HTTP requests) as well as the command description and help info can use the `symfony/console` styling format.
+
+```php
+namespace App\Cli\Command;
+
+use SilverStripe\PolyExecution\PolyCommand;
+use SilverStripe\PolyExecution\PolyOutput;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+
+class MyPolyCommand extends PolyCommand
+{
+    protected static string $commandName = 'app:my-command';
+
+    protected string $title = 'My command';
+
+    protected static string $description = 'A command that does something';
+
+    public function run(InputInterface $input, PolyOutput $output): int
+    {
+        if ($input->getOption('do-action')) {
+            $output->writeln('Doing something...');
+        }
+        return Command::SUCCESS;
+    }
+
+    public function getOptions(): array
+    {
+        return [
+            new InputOption('do-action', null, InputOption::VALUE_NONE, 'do something specific'),
+        ];
+    }
+}
+```
+
+> [!NOTE]
+> You can also optionally implement the static [`getHelp()`](api:SilverStripe\PolyExecution\PolyCommand::getHelp()) method to provide additional context about the command. This helps to keep the description short.
+
+Once you've set up your command in PHP, you can add an HTTP route to is using the regular [`Director` routing rules](/developer_guides/controllers/routing/), and add it to Sake with the [`Sake.commands`](api:SilverStripe\Cli\Sake->commands) configuration property. Note that this isn't necessary for the special subclasses mentioned in the sections below.
+
+```yml
+---
+Name: polycommands
+---
+SilverStripe\Control\Director:
+  rules:
+    my-command: 'App\Cli\Command\MyPolyCommand'
+
+SilverStripe\Cli\Sake:
+  commands:
+    - 'App\Cli\Command\MyPolyCommand'
+```
+
+This command could then be used by visiting `https://www.example.com/my-command` in a browser or running `sake app:my-command` in CLI.
+
+## Configuration
+
+For every `PolyCommand` subclass, including the special kinds listed below, you can set the following configuration:
+
+- [`can_run_in_cli`](api:SilverStripe\PolyExecution\PolyCommand->can_run_in_cli): Whether the command can be run in CLI
+- [`can_run_in_browser`](api:SilverStripe\PolyExecution\PolyCommand->can_run_in_browser): Whether the command can be run in HTTP requests
+- [`permissions_for_browser_execution`](api:SilverStripe\PolyExecution\PolyCommand->permissions_for_browser_execution): An array of permissions a user must have to run the command in an HTTP request
```

Comment:
```suggestion
- [`can_run_in_cli`](api:SilverStripe\PolyExecution\PolyCommand->can_run_in_cli): Whether the command can be run in CLI
- [`can_run_in_browser`](api:SilverStripe\PolyExecution\PolyCommand->can_run_in_browser): Whether the command can be run in HTTP requests
- [`permissions_for_browser_execution`](api:SilverStripe\PolyExecution\PolyCommand->permissions_for_browser_execution): An array of permissions of which the user must have at least one have to run the command in an HTTP request
```
By default `Permission::check()` will use `any` not `all`

Reply:
Clarified in a different way (based on the comment in the main issue I assumed there would be no suggestion here so I made the change before I saw this)